toolbox/commands.py

```diff
@@ -26,33 +26,27 @@ def _list_to_command_choices(
 
 
 @t.overload
-def as_command_choices(choices: t.Sequence[ChoiceTypes]) -> t.Sequence[hikari.CommandChoice]:
```

Comment:
Not sure why the formatting changed in this file, out of scope for this PR

Reply:
Will do when I get home